data/remotes.d/lvfs-testing.conf

```diff
@@ -7,6 +7,7 @@ MetadataURI=https://cdn.fwupd.org/downloads/firmware-testing.xml.@compression@
 ReportURI=https://fwupd.org/lvfs/firmware/report
 #Username=
 #Password=
-OrderBefore=lvfs,fwupd
+OrderBefore=lvfs,dell-esrt
```

Comment:
dell-esrt isn't installed in all configs.  This might need to be dynamic.

Reply:
If it's not installed you get a g_debug message -- hidden by default.

Reply:
Yeah but can't we also just use a template to build it dynamically?

Reply:
It's removed now :)